cpp/include/algorithms.h

```diff
@@ -25,35 +25,36 @@ namespace cugraph {
  * The number of iterations depends on the properties of the network itself; it increases when the tolerance descreases and/or alpha increases toward the limiting value of 1.
  * The user is free to use default values or to provide inputs for the initial guess, tolerance and maximum number of iterations.
  *
- * @Param[in] graph                  cuGRAPH graph descriptor, should contain the connectivity information as an edge list (edge weights are not used for this algorithm).
- *                                   The transposed adjacency list will be computed if not already present.
+ * @Param[in] graph                  cuGRAPH graph descriptor, should contain the connectivity information as a transposed adjacency list (CSR). Edge weights are not used for this algorithm.
  * @Param[in] alpha                  The damping factor alpha represents the probability to follow an outgoing edge, standard value is 0.85.
- Thus, 1.0-alpha is the probability to “teleport” to a random vertex. Alpha should be greater than 0.0 and strictly lower than 1.0.
- * @Param[in] has_guess              This parameter is used to notify cuGRAPH if it should use a user-provided initial guess. False means the user doesn't have a guess, in this case cuGRAPH will use a uniform vector set to 1/V.
- *                                   If the value is True, cuGRAPH will read the pagerank parameter and use this as an initial guess.
+                                     Thus, 1.0-alpha is the probability to “teleport” to a random vertex. Alpha should be greater than 0.0 and strictly lower than 1.0.
  *                                   The initial guess must not be the vector of 0s. Any value other than 1 or 0 is treated as an invalid value.
```

Comment:
I think the doc changed. This should be under has_guess definition

Reply:
Fixed. Good catch.